content/en/flux/use-cases/gh-actions-helm-promotion.md

```diff
@@ -5,6 +5,12 @@ description: "How to configure a promotion workflow for Flux HelmReleases with G
 weight: 31
 ---
 
+{{% alert color="warning" title="Disclaimer" %}}
```

Comment:
This guide was updated for Flux 2.0.0.

Reply:
OK, I fixed this in e2fdb17f905aca6c190e14eed746780883123ad8